website_animate/static/src/js/o_animate.frontend.js

```diff
@@ -5,16 +5,14 @@ var publicWidget = require('web.public.widget');
 
 var WebsiteAnimate = {
     win   : {},
-    items : {},
 
     offsetRatio : 0.3,  // Dynamic offset ratio: 0.3 = (element's height/3)
     offsetMin   : 10,   // Minimum offset for small elements (in pixels)
 
     // Retrieve animable elements and attach handlers.
     start: function () {
         var self   = this;
-        self.items = $(".o_animate");
```

Comment:
As a fix, keep this assignment (and its definition you removed above too) even if the variable is not used anymore. Otherwise, custo might be broken.